src/docs/development/data-and-backend/json.md

```diff
@@ -469,16 +466,20 @@ For more information, see [explicitToJson][] in the
 
 For more information, see the following resources:
 
-* [dart:convert][] and [JsonCodec][] documentation
-* [The
-  json_serializable package in Pub]({{site.pub}}/packages/json_serializable)
-* [json_serializable
-  examples in GitHub]({{site.github}}/dart-lang/json_serializable/blob/master/example/lib/example.dart)
-* [Discussion
-  about dart:mirrors in Flutter]({{site.github}}/flutter/flutter/issues/1150)
+* The [dart:convert][] and [JsonCodec][] documentation
+* The [json_serializable][] package on pub.dev.
```

Comment:
Delete the period at the end of this and the next item (or add periods after all of them)